zygo.json

```diff
@@ -6,11 +6,14 @@
       "bodyClass": "zygo"
     },
     "logo": "/assets/logo.png",
+    "home": "/zygo",
     "menu": {
-      "tutorials": "/category/tutorial",
-      "docs": "/category/docs",
-      "overview": "/post/overview",
-      "faq": "/post/faq"
+      "getting started": "/zygo",
+      "tutorials": "/tutorials/",
+      "docs": "/docs/",
+      "overview": "/overview",
+      "faq": "/faq",
+      "github": "https://github.com/zygo-io"
     }
   },
```

Comment:
Not sure if this sort of data should really be in the zygo.json? It should be wrappable with the installable routes.